its/src/test/java/com/sonar/it/scanner/msbuild/ScannerMSBuildTest.java

```diff
@@ -225,12 +226,46 @@ private void assertLineCountForProjectUnderTest(String projectKey) {
   @Test
   public void testExcludedAndTest_AnalyzeTestProject() throws Exception {
     int expectedTestProjectIssues = isTestProjectSupported() ? 1 : 0;
-    testExcludedAndTest(false, expectedTestProjectIssues);
+    String token = TestUtils.getNewToken(ORCHESTRATOR);
```

Comment:
here I copy-pasted because I needed the same thing except `.setProperty("sonar.dotnet.excludeTestProjects", ...` for my test